notebooks/254-llm-chatbot/config.py

```diff
@@ -18,6 +18,11 @@
 You are an assistant for question-answering tasks. Use the following pieces of retrieved context to answer the question. If you don't know the answer, just say that you don't know. Use three sentences maximum and keep the answer concise.\
 """
 
+DEFAULT_RAG_PROMPT_CHINESE = """\
```

Comment:
hi @yas-sim could you help to update the RAG prompt template for youri-7b-chat, thanks